src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -572,6 +576,7 @@
                            :fontSize "106%" :fontWeight 500 :textDecoration "none"
                            :color (:button-primary style/colors)}
                    :href "javascript:;"
+                   :data-test-id "cancel-button"
```

Comment:
In general, we should be cautious about static test IDs in reusable components, but it should be okay in this case because it's a modal and therefore there should only be one (unless there are stacked modals, but I think we may be okay there too because only the topmost one will be accessible? probably worth experimenting with that).